server/komodo-rest/src/main/java/org/komodo/rest/relational/response/RestConnectionSummary.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * See the COPYRIGHT.txt file distributed with this work for information
+ * regarding copyright ownership.  Some portions may be licensed
+ * to Red Hat, Inc. under one or more contributor license agreements.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301 USA.
+ */
+package org.komodo.rest.relational.response;
+
+import java.net.URI;
+
+import org.komodo.rest.RestBasicEntity;
+import org.komodo.rest.relational.connection.RestConnection;
+import org.komodo.spi.KException;
+
+/**
+ * An entity containing a connection and it's status.
+ */
+public final class RestConnectionSummary extends RestBasicEntity {
```

Comment:
A RestBasicEntity includes things like id and datapath so tends to represent an object from the komodo repo. To avoid that stuff with non-repo objects, extends AbstractKEntity instead (yes, naming conventions probably need to be updated).